### PR TITLE
Update select-plan.md

### LINKED ISSUE #1757: New Doc: Downgrading Site Plans
## Title

Downgrading Site Plans
## Description

Learn how to downgrade your site's plan and disable addon services such as Redis.
## Outline
- How to downgrade
- Considerations for downgrading to a sandbox (domains will be removed)
- How to disable Redis when downgrading to a personal plan
## Path

`/docs/downgrade-plan`

Surfaced by @fusionx1

### PR DESCRIPTION
**Pull request by lauren-kelly**
Closes #1757

## Effect
PR includes the following changes:
- Updated adding payment
- Added adjusting site plan, with warning re: domains and https
- Fixed broken links

## Remaining Work
- [ ] @ccorpus please review

### CODE REVIEW

**Comment by lauren-kelly** [comment]
@rachelwhitton this ought to be good to go - a lot got updated on it. @ccorpus would be a good one to review it, if you want a second pair of eyes on it.

File: source/_docs/select-plan.md [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+The following should be considered before downgrading your site from a paid plan to Sandbox:
+
+* All domains added to Pantheon environments will be removed. If you decide to return to a paid plan in the future, you will need to add the domains again in desired environments.
+* HTTPS will be disabled and certificates will be removed. If you decide to re-enable HTTPS in the future, you must re-upload your certificate, intermediary certificates, and your key.
+* A prorated credit will be issued to your billing account.
+
+### Downgrade or Upgrade to Personal
+The following should be considered before downgrading or upgrading to a Personal plan:
+
+* Redis and Solr will be disabled.
```
Comment (rachelwhitton) [inline source/_docs/select-plan.md]: @lauren-kelly I'd like to include steps to disable Redis and Solr within WordPress and Drupal. Can you help identify those steps?

**Comment by lauren-kelly** [comment]
@rachelwhitton I added some super basic info on redis/solr. If we find people need more info, I think we should maybe add a disable section to each enable doc - or new docs all together.

**Comment by lauren-kelly** [comment]
+1

**Pull request merged**